Source/WebCore/dom/NodeRareData.h

```diff
@@ -188,9 +188,9 @@ class NodeListsNodeData {
     inline void adoptDocument(Document& oldDocument, Document& newDocument);
 
 private:
-    std::pair<unsigned char, AtomString> namedCollectionKey(CollectionType type, const AtomString& name)
+    std::pair<uint8_t, AtomString> namedCollectionKey(CollectionType type, const AtomString& name)
     {
-        return std::pair<unsigned char, AtomString>(type, name);
+        return std::pair<uint8_t, AtomString>(static_cast<uint8_t>(type), name);
```

Comment:
Might be nice to use std::underlying_type

Reply:
Sure, we can do that.